build/phpDocumentor.sh

```diff
@@ -4,4 +4,5 @@ wget https://phpdoc.org/phpDocumentor.phar
 
 mkdir -p api/
 
-php7.4 phpDocumentor.phar -t "./api" -d "./lib/public" --title="Nextcloud PHP API ($BRANCH)"
+export PHP_VERSION=8.1
+php phpDocumentor.phar -t "./api" -d "./lib/public" --title="Nextcloud PHP API ($BRANCH)"
```

Comment:
```suggestion
php phpDocumentor.phar run -t "./api" -d "./lib/public" --title="Nextcloud PHP API ($BRANCH)"
```

Seems phpDocumentor changed their CLI